pom.xml

```diff
@@ -278,6 +279,23 @@
       <version>1.7</version>
     </dependency>
 
+    <!-- uv extension api -->
+    <dependency>
+      <groupId>de.focus-shift.urlaubsverwaltung.extension.api</groupId>
```

Comment:
hm, ist nun auch ok. Aber ich dachte irgendwie, wir packen das alles in eine depependency. Was hat das für Vorteile mehrere zu haben?

Reply:
die uv braucht alle. auf der extension-seite kann man dann z.b. nur einen event-listener implementieren und hat dann nur diese abhängigkeit und nicht die services.